src/components/document-content.tsx

```diff
@@ -61,14 +89,38 @@ export class DocumentContentComponent extends BaseComponent<IProps, {}> {
     return dataTransfer.types.findIndex(t => t === type) >= 0;
   }
 
+  private getDragResizeRowInfo(e: React.DragEvent<HTMLDivElement>) {
+    const rowId = extractDragResizeRowId(e.dataTransfer);
+    const startScreenY = extractDragResizeScreenY(e.dataTransfer);
+    const modelHeight = extractDragResizeModelHeight(e.dataTransfer);
+    const domHeight = extractDragResizeDomHeight(e.dataTransfer);
+    const deltaHeight = e.screenY - (startScreenY || 0);
```

Comment:
I'm a little unclear about the cases where we resize using the `modelHeight` vs. the `domHeight` - could you help clarify?

Reply:
Some tools (e.g. geometry) don't have an inherent content height, so we default-initialize them to 200px which gets stored in the model. Other tools (e.g. text and images) do have an inherent content height, and so we don't specify a particular height, preferring to let the browser size to the content. For these tools, height is undefined in the model, and so we have to base our resize decisions on the height of the DOM element. Once the height is changed, however, we store the height in the model, thus preventing the browser from sizing to content.

Reply:
Ahh awesome that makes sense, thanks Kirk! Is this line correct then?

```const rowHeight = dragResizeRow && (dragResizeRow.domHeight || dragResizeRow.modelHeight);```

Since it looks like it prefers `domHeight` over `modelHeight`.

Reply:
One might think so, but during a drag the `domHeight` is updating, but we only update the model once the drop is complete, so in the steady-state the DOM height should be the same as the model height and during drags it's more current.